ignite/pkg/cosmosgen/cosmosgen.go

```diff
@@ -103,9 +106,24 @@ func IncludeDirs(dirs []string) Option {
 	}
 }
 
+// UpdateBufModule enables Buf config proto dependencies update.
+// This option updates app's Buf config when proto packages or
+// Buf modules are found within the Go dependencies.
+func UpdateBufModule() Option {
+	return func(o *generateOptions) {
+		o.updateBufModule = true
+	}
+}
+
+// CollectEvents sets an event bus for sending generation feedback events.
+func CollectEvents(ev events.Bus) Option {
+	return func(c *generateOptions) {
+		c.ev = ev
```

Comment:
Maybe the events bus should be mandatory and not optional, wdyt?

Reply:
I didn't make it mandatory to allow using the generator in context where user feedback might not be required. That is not the case right now for the CLI but because `cosmosgen` is a package I thought that its API would be more flexible if we make it optional.